aspnetcore/host-and-deploy/health-checks/samples/2.x/HealthChecksSample/DBHealthStartup.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using HealthChecks.SqlServer;
```

Comment:
AddSqlServer extension method is on Microsoft.Extensions.DependencyInjection, this namespace entry is not necessary